Sources/RudderStackAnalytics/Policies/BackoffPolicy/BackoffPolicyHandler.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  BackoffPolicyHandler.swift
+//  RudderStackAnalytics
+//
+//  Created by Satheesh Kannan on 29/08/25.
+//
+
+import Foundation
+
+// MARK: - BackoffPolicyHandler
+/**
+ Actor is responsible for managing the backoff policy for retrying operations.
+ */
+final actor BackoffPolicyHandler {
+    private let maxAttempts: Int
+    private let coolOffSecs: Int
```

Comment:
Shall we consider renaming this variable to:
```suggestion
    private let coolOffPeriodInSecs: Int
```